Task-53253: Double delete action to delete a space group

wassimmhamdi opened the pull request:
Problem: The space group is deleted after clicking  twice on delete button
Fix: we set the broadcast event listener that it deletes a space without deleting the user's group. When a space is deleted, all of its page navigations and its group will be deleted and the listener get not started for a second time

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

houssemexo26 committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

Amine0008 committed:
Task-53233: Bad display of add space drawer for mobiles

ISSUE: the form to create a space overflow the border on the right for safari mobiles
FIX: added padding for this form

Amine0008 committed:
Task 45533: removed mobile module from social repo (#1232)

* removed modile module from social repo

* removed external config for the mobile protlet

Co-authored-by: Ali HAMDI <ahamdi@exoplatform.com>

Amine0008 committed:
Task 45533: removed mobile module from social repo (#1232)

* removed modile module from social repo

* removed external config for the mobile protlet

Co-authored-by: Ali HAMDI <ahamdi@exoplatform.com>

IlhemEssaadi committed:
TASK-53143: Import users from CSV Files, Non understandable message (#1251)

ISSUES : Import users from CSV Files, Non understandable message.
FIX : the problem fixed by changing the error message to be more understandable and returning the username.

IlhemEssaadi committed:
TASK-53143: Import users from CSV Files, Non understandable message (#1251)

ISSUES : Import users from CSV Files, Non understandable message.
FIX : the problem fixed by changing the error message to be more understandable and returning the username.

Commit:
Task-53253: Double delete action to delete a space group
Problem: The space group is deleted after clicking  twice on delete button
Fix: we set the broadcast event listener that it deletes a space without deleting the user's group. When a space is deleted, all of its page navigations and its group will be deleted and the listener get not started for a second time

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/Meeds-io/social/1263/333aafb0/dc16772fbab2ed6f2fc0f32cbf5cb786dca25cba.svg)](https://app.codesee.io/r/reviews?pr=1263&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=1263&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

IlhemEssaadi committed:
TASK-52005: Fix a new activity indicator that displayed in another space.

ISSUES : when creating a new post in a space, the new activity indicator is displayed in another space.
FIX : the problem that when creating a new message, the new activity indicator is displayed in another space and it is fixed by sending the spaceId of the space that the indicator should display in, in a websocket event and adding a condition allowing the display of the indicator in the relative space.

rdenarie reviewed (changes requested):
Can you fix the sonar feedback ? Thanks

houssemexo26 committed:
Merge branch 'feature/maintenance' into task-53253

rdenarie committed:
Merge branch 'feature/maintenance' into task-53253
